devtools/cli/src/main/java/io/quarkus/cli/Create.java

```diff
@@ -6,133 +6,92 @@
 import java.util.concurrent.Callable;
 
 import io.quarkus.cli.core.BaseSubCommand;
-import io.quarkus.cli.core.QuarkusCliVersion;
-import io.quarkus.devtools.commands.CreateProject;
-import io.quarkus.devtools.project.BuildTool;
-import io.quarkus.devtools.project.QuarkusProject;
-import io.quarkus.devtools.project.QuarkusProjectHelper;
-import io.quarkus.devtools.project.codegen.SourceType;
+import io.quarkus.cli.options.IncludeExamples;
+import io.quarkus.cli.options.TargetBuildTool;
+import io.quarkus.cli.options.TargetGAV;
+import io.quarkus.cli.options.TargetLanguage;
 import picocli.CommandLine;
 
-@CommandLine.Command(name = "create", sortOptions = false, mixinStandardHelpOptions = false, description = "Create a new quarkus project.")
+@CommandLine.Command(name = "create", sortOptions = false, showDefaultValues = true, mixinStandardHelpOptions = false, header = "Create a Quarkus project", description = "By default, this command will create a new 'code-with-quarkus' directory "
+        + "containing a Java project that will use maven to build a quarkus application with "
+        + "groupId='org.acme', artifactId='code-with-quarkus', and version='1.0.0-SNAPSHOT'.%n")
 public class Create extends BaseSubCommand implements Callable<Integer> {
```

Comment:
Last time I was involved in the discussion, I said `create` was not explicit enough as I have no idea what it creates. I think this still stands.

Especially since we will have `create-extension`.

I thought we agreed on this? Did you rediscuss this again?

Reply:
`create-app` is easy enough to amend. I can do that today and we can see what it feels like.

Reply:
+1 for create-app